pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/index/mutable/MutableForwardIndex.java

```diff
@@ -552,4 +552,48 @@ default void setDoubleMV(int docId, double[] values) {
   default void setStringMV(int docId, String[] values) {
     throw new UnsupportedOperationException();
   }
+
+  /**
+   * Reads the BYTES type multi-value at the given document id into the passed in value buffer (the buffer size must be
+   * enough to hold all the values for the multi-value entry) and returns the number of values within the multi-value
+   * entry.
+   *
+   * @param docId Document id
+   * @param valueBuffer Value buffer
+   * @return Number of values within the multi-value entry
+   */
+  default int getBytesMV(int docId, byte[][] valueBuffer) {
```

Comment:
(minor) Move the getters after the string getters

Reply:
Done